README.rst

```diff
@@ -302,6 +302,36 @@ Defaults to ``/etc/openwisp/post-registration-hook``;
 
 Path to an executable script that will be called after the registration is completed.
 
+Hotplug Events
+--------------
+
+*openwisp-config* will send hotplug events in the following situations:
```

Comment:
let's link hotplug events to https://openwrt.org/docs/guide-user/base-system/hotplug